mlflow/sklearn/utils.py

```diff
@@ -299,23 +299,24 @@ def _get_classifier_artifacts(fitted_estimator, prefix, X, y_true, sample_weight
 
     def plot_confusion_matrix(*args, **kwargs):
         import matplotlib
+        import matplotlib.pyplot as plt
 
         class_labels = _get_class_labels_from_estimator(fitted_estimator)
         if class_labels is None:
             class_labels = set(y_true)
 
         with matplotlib.rc_context(
             {
-                "figure.dpi": 288,
-                "figure.figsize": [6.0, 4.0],
-                "font.size": min(10.0, 50.0 / len(class_labels)),
-                "axes.labelsize": 10.0,
+                "font.size": min(8.0, 50.0 / len(class_labels)),
+                "axes.labelsize": 8.0,
+                "figure.dpi": 175,
             }
         ):
+            _, ax = plt.subplots(1, 1, figsize=(6.0, 4.0))
```

Comment:
This is what allows for the smaller font rendering within the labels of the confusion matrix.